src/order/ideal.lean

```diff
@@ -299,14 +336,76 @@ end
 
 end semilattice_sup_ideal_inter_nonempty
 
-section semilattice_sup_bot
-variables [semilattice_sup_bot P]
+section ideal_Inter_nonempty
+
+variables [preorder P] [ideal_Inter_nonempty P]
 
 @[priority 100]
-instance semilattice_sup_bot.ideal_inter_nonempty : ideal_inter_nonempty P :=
-{ inter_nonempty := λ _ _, ⟨⊥, ⟨bot_mem, bot_mem⟩⟩ }
+instance ideal_Inter_nonempty.ideal_inter_nonempty : ideal_inter_nonempty P :=
+{ inter_nonempty := λ _ _, begin
+    cases ideal_Inter_nonempty.exists_all_mem ‹_› with a ha,
+    exact ⟨a, ha _, ha _⟩
+  end }
+
+variables {α β γ : Type*} {ι : Sort*}
+
+lemma ideal_Inter_nonempty.ideal_Inter_nonempty {f : ι → ideal P} :
+  (⋂ x, (f x : set P)).nonempty :=
```

Comment:
The naming here is odd and doesn't indicate how this differs from `Inter_nonempty` above.

Reply:
This is analogous to `ideal_bInter_nonempty` and is a generalization of `Inter_nonempty`. Should I just remove it? How should it be renamed?